### ISSUE TITLE
Convert to more recent wrapping format

### ISSUE THREAD

**Issue by zouter** [huidongchen/TI_STREAM#1]
Hi huidong

I had a look at your wrapper for STREAM.

I assume you had some problems with getting conda installed inside the dynwrappy container (looking at https://github.com/huidongchen/TI_STREAM/blob/master/Dockerfile#L2) .

I think it should be possible to install conda yourself though using 

```Dockerfile
RUN wget --quiet https://repo.anaconda.com/miniconda/Miniconda2-4.5.11-Linux-x86_64.sh -O ~/miniconda.sh && \
    /bin/bash ~/miniconda.sh -b -p /opt/conda && \
    rm ~/miniconda.sh && \
    ln -s /opt/conda/etc/profile.d/conda.sh /etc/profile.d/conda.sh && \
    echo ". /opt/conda/etc/profile.d/conda.sh" >> ~/.bashrc && \
    echo "conda activate base" >> ~/.bashrc
```

BUT, I changed the format of containers for the competition yesterday, so that you don't need dynwrappy anymore to read and load in any data. You have to do the reading of HDF5 yourself though, but we have examples for R, Python and Julia to do this: https://github.com/dynverse/competition/blob/master/docs/description.md#quick-start

So could you as an excercise try to convert the ti_stream docker into this new "simplified" wrapping method? If you run into any problems, feel free to give me some feedback!

Thanks

**Comment by huidongchen** [comment]
Hi Wouter,

Thanks for taking time to help go over STREAM script! My apologies for the late reply. Somehow I didn't get notification email and I'm just seeing it now. 

But the good thing is the issue should be fixed now :) . i have re-structured the scripts. I'm able to build the submission docker and also managed to run the docker command line on example datasets. I also built the TC scorer docker successfully although I am having some trouble scoring the results. But I should be able to figure it out soon. Meanwhile ,please also let me know if you there is any part i should modify in the current wrapping method.

Thanks!

Huidong

**Comment by zouter** [comment]
Thanks huidong! I fixed a tiny part, see my pull request #2 for that. But I also managed to get the docker working and I am able to score it! :+1:
